androidApp/src/main/java/press/IntentReceiverActivity.kt

```diff
@@ -0,0 +1,20 @@
+package press
+
+import android.app.Activity
+import android.os.Bundle
+import press.editor.EditorActivity
+
+class IntentReceiverActivity : Activity() {
```

Comment:
"Intent receiver" is a bit too generic. Let's rename this to `LauncherShortcutActivity`?